app/src/main/java/com/example/mykotlincalculator21/mvp/presenter/CalculatorPresenter.kt

```diff
@@ -18,7 +19,21 @@ class CalculatorPresenter(
     }
 
     override fun operatorResultPressed() {
-        view.drawNumber(model.getFullResult())
+        model.doOperations()
+        when (model.getError()) {
+            ErrorUtils.NONE -> {
```

Comment:
Remember that we talked about change the name of ErrorUtils for ResultUtils and you can replace the NONE by a SUCCESS

Reply:
Thx for the tip, now its changed